server.azuredeploy.json

```diff
@@ -526,7 +526,7 @@
         "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'),'/extensions/joindomain')]",
         "[resourceId('Microsoft.Compute/virtualMachines/extensions',  variables('vmName'), variables('encryptionExtensionName'))]",
         "[concat('Microsoft.Resources/deployments/', variables('vmName'), '-encryption-update')]",
-        "[concat('Microsoft.Compute/virtualMachines/extensions/', variables('vmName'), '/ChefClient')]"
+        "[concat('Microsoft.Compute/virtualMachines/extensions/', variables('vmName'), '/extensions/ChefClient')]"
```

Comment:
[concat('Microsoft.Compute/virtualMachines/', variables('vmName'), '/extensions/', variables('vmName'), 'ChefClient')]

Reply:
Fixed

Reply:
the intended path would be
Microsoft.Compute/virtualMachines/CHEFSERVERNAME/extensions/CHEFSERVERNAME/ChefClient